src/main/resources/locales/en/translation/steps-to-making-your-claim.json

```diff
@@ -1,10 +1,67 @@
 {
   "title": "Steps to making your claim",
-  "h1": "Steps to making your claim (to do)",
-  "section1": {
-    "title": "1. Your details",
-    "link1Txt": "Personal details",
-    "link2Txt": "Contact details",
-    "link3Txt": "Your preferences"
-  }
+  "h1": "Steps to making your claim",
+  "sections": [
+    {
+      "title": "1. Your details",
+      "links": [
+        {
+          "url": "/dob-details",
```

Comment:
don't need to define urls here, PageUrls object is already passed into the base view template, can just use the constant values provided there in the page template, ensures future maintenance if we change one of the routes it gets updated everywhere.

Reply:
can put the name of the property here then add a method in the global nunjuck index file to resolve the url, but idk if that's making it too complicated.....